images/builder/docker/custom-docker-builder/Dockerfile

```diff
@@ -14,7 +14,7 @@
 #
 # The standard name for this image is openshift/origin-custom-docker-builder
 #
-FROM openshift/origin-base
+FROM openshift/origin-base:v1.5
```

Comment:
As I understand it, `FROM` stanza is updated in dist-git before images are built by OSBS. These files aren't taken verbatim to build. You'll notice they are also missing the OSBS-specific labels